src/frontend/app/features/applications/application/application-tabs-base/tabs/github-tab/github-tab.component.html

```diff
@@ -0,0 +1,93 @@
+<app-tile-grid class="github-tab" *ngIf="stratosProject$ | async as stratosProject">
+<app-tile-group>
+  <app-tile>
+    <mat-card>
+      <mat-card-header>
+        <mat-card-title>Deployment Information</mat-card-title>
+      </mat-card-header>
+      <mat-card-content>
+        <div class="github-tab__deployment" >
+            <app-metadata-item label="Repository">
+              <a href="{{(gitHubRepo$ | async)?.html_url}}" target="_blank">
+                {{stratosProject.deploySource.project }}
+              </a>
+            </app-metadata-item>
+            <app-metadata-item  label="Branch">
+              <a href="{{(gitHubRepo$ | async)?.html_url}}/tree/{{ stratosProject.deploySource.branch}}" target="_blank">
+                {{ stratosProject.deploySource.branch}}
+              </a>
+            </app-metadata-item>
+            <div class="github-tab__deployment__commit">
+              <div>
+                  <app-metadata-item  label="Commit">
+                      <a href="{{(commit$ | async)?.html_url}}" target="_blank">
+                          {{ stratosProject.deploySource.commit}}
+                      </a>
+                    </app-metadata-item>
+              </div>
+              <div class="github-tab__deployment__commit-warning" *ngIf="!(isHead$ | async)">
+                    <div><mat-icon>warning</mat-icon> </div>
+                    <div class="github-tab__deployment__commit-warning__msg">The branch {{ stratosProject.deploySource.branch}} has been updated since this app was pushed. Redeploy to update application!</div>
```

Comment:
Could read something like.. `The master branch has been updated since the app was pushed. Redeploy to update`